docs/install.rst

```diff
@@ -24,3 +24,10 @@ You can setup an app in the `Google App Engine <https://developers.google.com/ap
 by changing the ``application`` key in app.yaml to your application name. 
 For using it you just need to do the API calls to :samp:`http://{your-app-name}.appspot.com`.
 It needs some external python modules in :samp:`lib`, you can download them by running :command:`./devscripts/setup-gae.sh`
+
+Heroku
+******
+You can also run the server on `Heroku <https://heroku.com>`_ by adding ``gunicorn`` to ``requirements.txt``
+and creating a ``Procfile`` with ``web: gunicorn --pythonpath youtube_dl_server app:app -w 4 --log-file -``.
```

Comment:
Hmm, you're right! 

In <https://devcenter.heroku.com/articles/python-gunicorn#basic-configuration>:
```
The WEB_CONCURRENCY environment variable is automatically set by Heroku, based on the 
processes' Dyno size. This feature is intended to be a sane starting point for your 
application. We recommend knowing the memory requirements of your processes and setting 
this configuration variable accordingly.
```

So I guess it's best that we leave `-w` out. 😀